CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ option(WASMEDGE_PLUGIN_PROCESS "Enable and build WasmEdge process plugin." OFF)
 option(WASMEDGE_PLUGIN_STABLEDIFFUSION "Enable and build WasmEdge stable-diffusion plugin." OFF)
 option(WASMEDGE_PLUGIN_STABLEDIFFUSION_CUBLAS "Enable CUBLAS in the stable-diffusion plugin." OFF)
 option(WASMEDGE_PLUGIN_STABLEDIFFUSION_METAL "Enable Metal in the stable-diffusion plugin." OFF)
+option(WASMEDGE_PLUGIN_STABLEDIFFUSION_OPENMP "Enable OpenMP in the stable-diffusion plugin." ON)
```

Comment:
The default value should be `OFF` since most of the use cases are the GPU, which doesn't need the OpenMP.